app/assets/stylesheets/ama_layout/layout_components/callouts.scss

```diff
@@ -56,13 +56,11 @@
 }
 
 .promo-block{
-  @include callout($ash);
```

Comment:
removing the callout affects the registries promo-block also. I would suggest keeping them both consistent (with the grey bg), or creating a modifier of no-bg.